server_extensions_extra/src/main/resources/batm-extensions.xml

```diff
@@ -822,6 +822,51 @@
         <cryptologo cryptocurrency="DOGE" file="doge.png"/>
         <cryptologo cryptocurrency="DOGE" file="doge.svg"/>
     </extension>
+    <extension class="com.generalbytes.batm.server.extensions.extra.newyorkcoin.NewYorkCoinExtension" >
+        <wallet prefix="newyorkcoind" name="Newyorkcoind">
+            <tunnel supported="true"/>
+            <sendtomany supported="true"/>
+            <param name="protocol" />
+            <param name="user" />
+            <param name="password" />
+            <param name="host" />
+            <param name="port" />
+            <param name="label" />
+            <cryptocurrency>NYC</cryptocurrency>
+        </wallet>
+        <wallet prefix="Newyorkcoindnoforward" name="Newyorkcoind (no forward)">
+            <tunnel supported="true"/>
+            <sendtomany supported="true"/>
+            <param name="protocol" />
+            <param name="user" />
+            <param name="password" />
+            <param name="host" />
+            <param name="port" />
+            <cryptocurrency>NYC</cryptocurrency>
+            <help>This variant generates new addresses for each transaction and does not forward payments from temporary address</help>
+        </wallet>
+        <ratesource prefix="nycfix" name ="Fix Price" >
+            <param name="price" />
+            <cryptocurrency>NYC</cryptocurrency>
+            <help>USD is default fiat currency.</help>
+        </ratesource>
+        <ratesource prefix="CoinPaprikaRateSource" name ="CoinPaprika Rate Source for NYC" >
+            <param name="fiatCurrency" />
+            <cryptocurrency>NYC</cryptocurrency>
+            <cryptocurrency>BTC</cryptocurrency>
+            <cryptocurrency>LTC</cryptocurrency>
+            <cryptocurrency>ETH</cryptocurrency>
+        </ratesource>
+        <ratesource prefix="CoinGeckoRateSource" name ="CoinGecko Rate Source for NYC" >
+            <param name="fiatCurrency" />
+            <cryptocurrency>NYC</cryptocurrency>
+            <cryptocurrency>BTC</cryptocurrency>
+            <cryptocurrency>LTC</cryptocurrency>
+            <cryptocurrency>ETH</cryptocurrency>
+        </ratesource>
```

Comment:
Please add NYC to the existing rate sources, do not duplicate them here, thanks

Reply:
ahh ok hold on, let me fix that real quick, sorry about that